vsketch/sketch_class.py

```diff
@@ -136,6 +160,27 @@ def set_param_set(cls, param_set: dict[str, Any]) -> None:
     def param_set(self) -> dict[str, Any]:
         return {name: param.value for name, param in self._params.items()}
 
+    def get_future_path(self) -> pathlib.Path | None:
+        """Returns a :class:`Path` object pointing to the future location
+        of the exported file when 'like' is pressed, or `None` if viewer
+        is not set.
+        """
+        if self._viewer is None:
+            return None
+        else:
+            return self._viewer.get_future_path()
```

Comment:
It is not possible to know what the future path is, as it depend on whether the user use the `Like` button or `vsk save`. I suggest removing that altogether.